tests/functional/manage/test_views.py

```diff
@@ -48,6 +53,69 @@ def test_save_account(self, pyramid_services, user_service, db_request):
         assert user.name == "new name"
         assert user.public_email is None
 
+    def test_changing_password_succeeds(self, webtest, remote_addr, socket_enabled):
+        """If the user changes their password, their session should be invalidated."""
+        totp_known_secret = b"\x453f47ec8c43d1f205f0a5244391342b428b3b06"
+        # create a User
+        user = UserFactory.create(
+            with_verified_primary_email=True,
+            password=(  # 'password'
+                "$argon2id$v=19$m=1024,"
+                "t=6,p=6$EiLE2Nsbo9S6N+acs/beGw$ccyZDCZstr1/+Y/1s3BVZ"
+                "HOJaqfBroT0JCieHug281c"
+            ),
```

Comment:
TODO: I think the UserFactory should be able to take in a string input and generate the appropriate hashed value

Reply:
Something like `unhashed_password="password"` would be nice, but I think we could also just make this the default value.

Reply:
Added in 817819cad0fb405e25b86748e9e124ad2aa9a18a